epoch/api/chain_ws_api.md

```diff
@@ -306,6 +266,38 @@ Event:
 
 ```
 
+## Added micro block
+### Request
+ * **target:** `chain`
+ * **action:** `subscribe`
+ * **payload:**
+
+  | Name | Type | Description | Required |
+  | ---- | ---- | ----------- | -------- |
+  | type | string | `added_micro_block` | Yes |
```

Comment:
I see this in https://github.com/aeternity/epoch/pull/1293

Who is meant to use this? It may be beneficial to get their review of what they expect in response (may be separate future PR).

Reply:
I suppose it can be useful for blockchain explorer or maybe the apps team.